premake5.lua

```diff
@@ -74,10 +74,10 @@ project "rlExtrasC"
 	includedirs { "raylib/src","rlExtrasC"}
 	vpaths 
 	{
-		["Header Files"] = { "rlExtrasC/**.h"},
+		["Header Files"] = { "rlExtrasC/**.h", "raylib/src/raylib.h"},
 		["Source Files"] = {"rlExtrasC/**.c"},
 	}
-	files {"rlExtrasC/**.h", "rlExtrasC/**.c"}
+	files {"rlExtrasC/**.h", "rlExtrasC/**.c", "raylib/src/raylib.h"}
```

Comment:
These are to add the header file to the project for editing. They are not needed for include dirs. I would prefer to not directly add raylib.h to the project and just let the include dirs handle finding it. More than just raylib.h is used.